content/android/_index.md

```diff
@@ -11,4 +11,8 @@ listdescription: ""
 
 ### Overview
 
-Some overview on the Android project...
+WorldWind Android is a 3D virtual globe API for Android. You can use it to provide a geographic context, complete with terrain, for visualizing geographic or geo-located information.
+
+WorldWind Android provides high-resolution terrain and imagery, retrieved from remote servers automatically as needed. You can also provide your own terrain and imagery. WorldWind Android additionally provides a rich collection of shapes that you can use to represent information on the globe or in space.
+
+The WorldWind Android client is compatible to [API level](https://source.android.com/source/build-numbers) 19 (KitKat 4.4) and up. It displays the globe via the [GLSurfaceView](https://developer.android.com/reference/android/opengl/GLSurfaceView.html). Read more about how to integrate WorldWind into your app in the [Get Started](/android/get-started/) section. See how to implement basic functionality in the [Tutorials](/android/tutorials/) and dive into more topics in the [Developer Guide](/android/developer-guide/).
```

Comment:
If the site is complete as-is then we can publish it sooner rather than later. That's a good thing, and it's what drove me to suggest that we suppress unused links until they point to content. 

With regard to tutorials, I recommended that we split part of the Get Started page into our first tutorials. If we decide to do that, Tutorials would have some content.

Attempting to keep the navigation bar templates consistent across projects makes sense, until we find a need to customize.